pandas-iris/README.md

```diff
@@ -1,11 +1,14 @@
-# The `pandas-iris` Kedro starter
+# [ARCHIVED] This starter is no longer maintained.
+You can still find the starter code in older release tags: https://github.com/kedro-org/kedro-starters/tree/0.18.14/pandas-iris 
 
-## Introduction
+## The `pandas-iris` Kedro starter
 
-The code in this repository demonstrates best practice when working with Kedro. It contains a Kedro starter template with some initial configuration and an example pipeline, and originates from the [Kedro Iris dataset example](https://docs.kedro.org/en/stable/get_started/new_project.html).
+### Introduction
 
+The code in this repository demonstrates best practice when working with Kedro. It contains a Kedro starter template with some initial configuration and an example pipeline, and originates from the [Kedro Iris dataset example](https://docs.kedro.org/en/0.18.14/get_started/new_project.html).
 
```

Comment:
I feel like we should explain how to use this starter here since it's now archived. I will include something in the main starters docs but possibly drop in something like this?

This starter is unavailable in Kedro version 0.19.0 and beyond and the latest version of Kedro that supports it is Kedro 0.18.14. To check the version of Kedro you have installed, type `kedro -V` in your terminal window. To install a specific version of Kedro, e.g. 0.18.14, type `pip install example-package<=0.18.14`.

To create a project with this starter using `kedro new`,  type the following (assuming Kedro version 0.18.14):

```bash
kedro new --starter=pandas-iris --checkout=0.18.14
```

Reply:
I've updated this for all archived starters.